afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+smøre	-2
+smil	2
+smilte	2
+smiler	2
+smilende	2
+smugle	-2
+smuglet	-2
+smugling	-2
+smugler	-2
+sleipe	-1
+nyse	-2
+nøs	-2
+nyser	-2
+nysing	-2
+trykkrør	-2
+tankevekkende	1
+høytidelig	-1
+fast	2
+solidaritet	2
+størknet	2
+stivner	2
+størkne	2
+løsning	1
+løsninger	1
+en type	0
+son-of-a-tispe	-5
+sofistikert	2
+sår	-1
+sorgfull	-2
+beklager	-1
+romslig	1
+spammere	-3
+gnist	1
+gnisten	3
+glitrer	3
+musserende	3
+spydspiss	2
+spekulativ	-2
+ånd	1
+spirit	-2
+ondskapsfull	-2
+bortskjemte	-2
+plettfrie	2
+struttende	2
+sløse	-2
+sløst bort	-2
+sløsing	-2
+stikke	-2
+knivstukket	-2
+stabil	2
+stoppet	-1
+steile	-2
+utholdenhet	2
+stinket	-2
+oppsiktsvekkende	3
+sulte	-2
+sultet	-2
+sulter	-2
+standhaftig	2
+stjele	-2
+stjeler	-2
+stereotype	-2
+kvalt	-2
+stimulere	1
+stimulert	1
+stimulerer	1
+stimulerende	2
+stikkende	-2
+stinke	-2
+stinkende	-2
+stinker	-2
+stjal	-2
+stjålet	-2
+stoppe	-1
+stoppe	-1
+stopp	-1
+traust	2
+rett	1
+rar	-1
+styrke	2
+forsterke	2
+styrking	2
+styrker	2
+understreke	-1
+stresset	-2
+streik	-1
+sterk	2
+sterkere	2
+sterkeste	2
+streve	-2
+slet	-2
+sliter	-2
+sittende fast	-2
+slående	4
+dumhet	-3
+beleven	2
+stevning	-2
+betydelig	1
+i det vesentlige	1
+undergravende	-2
+lykkes	3
+lyktes	3
+suksess	2
+vellykket	3
+suge	-3
+suger	-3
+saksøke	-2
+saksøkt	-2
+sues	-2
+lide	-2
+led	-2
+lidende	-2
+lider	-2
+selvmord	-2
+velegnet	2
+mutt	-2
+solskinn	2
+suveren	5
+overlegen	2
+brukerstøtte	2
+støttet	2
+støttespillere	1
+bære	1
+støttende	2
+støtter	2
+overlevde	2
+overlevende	2
+mistenkt	-1
+mistanke	-1
+mistenkte	-1
+henge	-1
+suspendert	-1
+mistenkelig	-2
+bærekraft	1
+bærekraftig	2
+sverge	-2
+banning	-2
+sverger	-2
+søtere	3
+søteste	3
+fort	2
+raskt	2
+svindle	-3
+svindlerier	-3
+sympati	2
+plett	-2
+skjemt	-2
+tard	-2
+anløpe	-2
+anløpet	-2
+oksiderer	-2
+tårer	-2
+mør	2
+ømhet	2
+anspent	-2
+spenning	-1
+veldig bra	4
+livredd	-3
+skrekk	-3
+terrorister	-2
+terror	-3
+terrorisert	-3
+terroriserer	-3
+takke	2
+takk	2
+tornete	-2
+omtenksom	2
+true	-2
+truer	-2
+begeistret	5
+forpurret	-2
+kneble	-2
+toftene	-2
+engstelige	-2
+trett	-2
+puppene	-2
+toleranse	2
+tannløs	-2
+topp	2
+topper	2
+revet	-2
+tortur	-4
+torturert	-4
+torturere	-4
+totalitær	-2
+totalitarisme	-2
+spioner	-2
+giftig	-3
+tragedier	-2
+tragedie	-2
+tragisk	-2
+overtre	-2
+trådt	-2
+felle	-1
+fanget	-2
+feller	-1
+traume	-3
+traumatisk	-3
+parodi	-2
+forræderisk	-3
+skatt	2
+skatter	2
+skjelving	-2
+skjelvinger	-2
+hyllest	2
+lureri	-2
+triumf	4
+triumferende	4
+problema	-2
+ekte	2
+tillit	1
+klarert	2
+stiftelser	1
+svulst	-2
+tyran	-3
+tyranniske	-3
+tyrannisk	-3
+tyrans	-3
+allestedsnærværende	2
+stygghet	-3
+stygg	-3
+ute av stand	-2
+uakseptabelt	-2
+uattraktiv	-2
+utilgjengelig	-1
+uutholdelig	-2
+ikke til å tro	-1
+objektivt	2
+uklar	-2
+ubekreftet	-1
+usikre	-2
+kokt	-2
+undervurder	-1
+undervurdert	-1
+underestimerer	-1
+underslår	-1
+undergrave	-2
+undergraves	-2
+undergraver	-2
+tapte terreng	-2
+dårlige	-2
+dårlige resultater	-2
```

Comment:
This seems superfluous given the one above

Reply:
I'll remove this..